hardware/RF-Test-Guide.md

```diff
@@ -0,0 +1,283 @@
+ <img src="files\HW-RF-Test-Guide\BER2.png">  
+</div>
+
+After build the Railtest tool, you can use the commander tool to program customer device, then the device will be ready for RF test.
+More information for Simplicity Studio usage, please visit the application note [AN0822](https://www.Silabs.com/documents/public/application-notes/AN0822-simplicity-studio-user-guide.pdf)
+
+## 3.3. Tune and measure
+The following strategy is proposed to determine the optimal Ctune value:
+- Performe a conducted measurement;
+- Connect the custom board's RF port to a Spectrum Analyzer(SA);
+- Use SA to measure the fundamental frequency as following picture shows:
+  <div align="center">
+  <img src="files\HW-RF-Test-Guide\CTune-CW.png">  
+  </div>
+- Make sure to set appropriate span and Resolution Bandwitdth (RBW) on the spectrum analyzer. Use a few MHz frequency range as span and a few kHz as RBW. 
+- Tune the CTune value until the frequency of the fundamental harmonic reach the optimal value. 
+- Use RailTest tool to tune CTune. The RailTest commands are listed below: 
+  - :>rx 0  
+  - :>SetPower [decidBm]  
+  - :>setdebugmode 1  
+  - :>freqoverride 868000000  
+  - :>GetCTune  
+  - :>SetCTune 0x[hex-value] or [desimal value] 
+  - :>SetTxtone 1  
+  - :>SetTxtone 0
+
+Railtest commands explaination:
+- Railtest starts with "rx 0" command. 
+- The actual output power can be set by "SetPower" command.
+- The actual output frequency can be set by "setdebugmode 1" and "freqoverride"  commands.
+- The actual CTune variable can be read by "GetCtune" command. 
+- It can set a new value to CTune, which can be done by "SetCTune" command. 
+- After the configuration, the "SetTxTone 1" command sets the radio to transmit mode. In this case the radio transmits CW signal. 
+- The CW transmission can be disabled by "SetTXtone 0" command. 
+
+For more Railtest commands information, please refer to this [KBA](https://www.Silabs.com/community/wireless/proprietary/forum.topic.html/railtest_-_more_info-66AE).
+
+For more CTune calibration details, it is discussed in this [KBA.](https://www.Silabs.com/community/wireless/proprietary/knowledge-base.entry.html/2019/03/18/hfxo_capacitor_bank-7uRt)
+
+
+# 4. Contucted Tx fundamental power and harmonics measurement
+The fundamental power and harmonics measurement are the basic test items for Tx performance. Usually The test will deploy a single tone signal at its maximum 
+output power state, then the CW signal will be injected into a SA by a RF cable for the measurement. The RF cable should be attached to the 50 ohm point, and the antenna and its matching circuits should be removed from the RF cable attaching point. For generating the desire CW signal, it can use the Railtest tool to control the radio board to do that. As the Railtest tool is already generated in chapter 3. It can be used for this test directly.
+## 4.1. RailTest commands for Tx CW signal
+The RailTest commands are listed below: 
+  - :>rx 0
```

Comment:
Are the :> on each line of this bullet list required or can they be removed?

Reply:
Changed " :>" to "->".